src/item/components/modals/ReportItemModal.tsx

```diff
@@ -0,0 +1,191 @@
+import { get } from 'lodash-es';
+import React, { FunctionComponent, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+	Button,
+	ButtonToolbar,
+	FormGroup,
+	Modal,
+	ModalBody,
+	ModalFooterRight,
+	RadioButton,
+	RadioButtonGroup,
+	Spacer,
+	Spinner,
+	TextArea,
+	Toolbar,
+	ToolbarItem,
+	ToolbarRight,
+} from '@viaa/avo2-components';
+import { Avo } from '@viaa/avo2-types';
+
+import toastService from '../../../shared/services/toast-service';
+import { ZendeskService } from '../../../shared/services/zendesk-service';
+import i18n from '../../../shared/translations/i18n';
+
+interface ReportItemModalProps {
+	externalId: string;
+	isOpen: boolean;
+	onClose: () => void;
+	user: Avo.User.User;
+}
+
+const RADIO_BUTTON_LABELS = {
+	broken: i18n.t('Defect'),
+	inappropriate: i18n.t('Ongepast '),
+	copyright: i18n.t('Schending auteursrechten'),
+};
+
+const ReportItemModal: FunctionComponent<ReportItemModalProps> = ({
+	externalId,
+	isOpen,
+	onClose,
+	user,
+}) => {
+	const [t] = useTranslation();
+
+	const [reason, setReason] = useState<keyof typeof RADIO_BUTTON_LABELS | null>(null);
+	const [extraDetails, setExtraDetails] = useState<string>('');
+	const [isProcessing, setISProcessing] = useState<boolean>(false);
+
+	const reportItem = async () => {
+		let ticket: any = undefined;
```

Comment:
In dit geval lijkt dit me dezelfde vorm te hebben en dus bepaalbaar qua type? Of ben ik verkeerd?